tests/integration/api_v2/http/delegates.test.js

```diff
@@ -0,0 +1,142 @@
+const endpoint = `${baseUrlV2}/accounts`;
+
+describe('Delegates API', () => {
+	let refDelegate;
+	beforeAll(async () => {
+		[refDelegate] = (await api.get(`${endpoint}?isDelegate=true&limit=1`)).data;
+	});
+
+	describe('GET /accounts?isDelegate=true', () => {
+		it('known delegate by address -> ok', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&address=${refDelegate.summary.address}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeArrayOfSize(1);
+			response.data.forEach(account => {
+				expect(account).toMap(accountSchemaVersion5);
+				expect(account.dpos).toMap(dpos);
+			});
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('wrong input param -> 400', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&id=412875216073141752800000`, 400);
+			expect(response).toMap(wrongInputParamSchema);
+		});
+
+		// TODO Implement support for search in accounts
+		xit('search delegates -> ok', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&search=genesis`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeInstanceOf(Array);
+			expect(response.data.length).toBeGreaterThanOrEqual(1);
+			response.data.forEach(account => {
+				expect(account).toMap(accountSchemaVersion5);
+				expect(account.username).toContain('genesis');
+			});
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('invalid address -> 400', async () => {
+			const response = await api.get(`${endpoint}?address=412875216073141752800000`, 400);
+			expect(response).toMap(badRequestSchema);
+		});
+	});
+
+	describe('GET /accounts?isDelegate=true&publickey', () => {
+		it('known delegate by public key -> ok', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&publickey=${refDelegate.summary.publicKey}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeArrayOfSize(1);
+			response.data.forEach(account => {
+				expect(account).toMap(accountSchemaVersion5);
+				expect(account.dpos).toMap(dpos);
+			});
+			expect(response.meta).toMap(metaSchema);
+		});
+
+		it('wrong delegate publickey -> 400', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&publickey=412875216073141752800000`, 400);
+			expect(response).toMap(badRequestSchema);
+		});
+	});
+
+	describe('GET /accounts?isDelegate=true&username', () => {
+		it('known delegate by username -> ok', async () => {
+			const response = await api.get(`${endpoint}?isDelegate=true&username=${refDelegate.summary.username}`);
+			expect(response).toMap(goodRequestSchema);
+			expect(response.data).toBeArrayOfSize(1);
+			response.data.forEach(account => {
+				expect(account).toMap(accountSchemaVersion5);
+				expect(account.dpos).toMap(dpos);
+			});
+			expect(response.meta).toMap(metaSchema);
+		});
+	});
+	// Fix delegate search by status
+	xdescribe('GET /accounts?isDelegate=true&status', () => {
```

Comment:
We should be good to enable this test suite now that we support status based delegate search.
Also, please make sure to begin a `TODO` task with the keyword. This would help us to quickly search for any such notes.

```suggestion

	describe('GET /accounts?isDelegate=true&status', () => {
```

Reply:
Search delegates by status is not supported in SDKv5 right now. I had discussion with @MichalTuleja today and I can do it with issue #261 and enable the tests.